### PR TITLE
Beatmap Discussion: update from forum thread again (again)

### PR DESCRIPTION
**Pull request by MegaApplePi**
from <https://osu.ppy.sh/community/forums/topics/650961?n=1>

### CODE REVIEW

File: wiki/Beatmap_Discussion/en.md [hunk 1]
```diff
@@ -38,11 +38,11 @@ Clicking on `Modding/Discussion` on the beatmap's page will take you to the page
 
 Changes the game mode between osu!standard, osu!taiko, osu!catch, and osu!mania. You can only select each game mode if the beatmapset has a difficulty with said game mode.
 
-## Beatmap page / Difficulty selection
+### Beatmap page / Difficulty selection
 
 ![](img/difficulty-menu.jpg "Difficulty menu")
 
-Clicking on the Artist/Title of the beatmap will take you back to the Beatmap Information page. You can select a difficulty to mod using the dropdown menu. **Make sure you are on the correct difficulty before submitting mods to the beatmap!**
+Clicking on the Artist/Title of the beatmap will take you back to the Beatmap Information page. You can select a difficulty to mod using the dropdown menu. The numbers that appear besides the difficulty name are the number of pending discussions it currently has to resolve. **Make sure you are on the correct difficulty before submitting mods to the beatmap!**
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: The number that appear
it currently has to resolve -> which currently need to be resolved?

File: wiki/Beatmap_Discussion/en.md [hunk 2]
```diff
@@ -82,9 +82,11 @@ Hype train is similar to the old star priority system. A user can submit Praise
 
 *Note: the creator can't Hype up their own beatmap. Praise must be done in the General (All Difficulties) section to add Hype. It will not work through the General or the Timeline tabs.*
 
+Hype must be done in the General (All Difficulties) section to add Hype. It will not work through the General or the Timeline tabs. Players are required to play through a difficulty in the beatmapset in order to add Hype to a Beatmap.
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: Hype must be done in the General (All Difficulties) section to add Hype -> If you want to add Hype, it must be done in the General (All Difficulties) section.

"must be done in the General section" "it wont work through the General tab" what the fuck
why is a "beatmap" capitalized at the end of the line

File: wiki/Beatmap_Discussion/en.md [hunk 3]
```diff
@@ -111,13 +113,17 @@ This is where you type your mod. If you use the Timeline section, you must inclu
...
 ### Mod type
 
 ![](img/mod-type.jpg)
 
-This is where you submit your mod. You can choose between Praise, Suggestion, or Problem. Each is fairly self-explanatory. Clicking on this will submit your mod. If you made any mistake or need to make an edit to your mod, find the post in the Discussion section.. You can add Hype to the beatmap by submitting Praise in the General (All Difficulties) section.
+This is where you submit your mod. You can choose between Praise, Suggestion, or Problem. Each is fairly self-explanatory. **Clicking on this will submit your mod. If you made any mistake or need to make an edit to your mod, find the post in the Discussion section.** You can add Hype to the beatmap by submitting Praise in the General (All Difficulties) section.
+
+### Pin
+
+A blue button with a pin icon. Appears to do nothing at this time.
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: at the time

File: wiki/Beatmap_Discussion/en.md [hunk 4]
```diff
@@ -72,19 +72,21 @@ This contains the creator who submitted the beatmapset, shows the submission dat
...
 > Adding praise ❤ will raise this beatmap's hype, making it more visible for nomination and ranking!
 
 *Note: the creator can't Hype up their own beatmap. Praise must be done in the General (All Difficulties) section to add Hype. It will not work through the General or the Timeline tabs.*
 
+Hype must be done in the General (All Difficulties) section to add Hype. It will not work through the General or the Timeline tabs. Players are required to play through a difficulty in the beatmapset in order to add Hype to a Beatmap.
+
 ### Nomination
 
-Once a beatmap has accumilated 12 Hype, the beatmap is eligible for Nomination. This works as it did with the old modding system. The beatmap is required to have 2 nominations by the [Beatmap Nominators](/wiki/Beatmap_Nominators) to move into the qualified state.
+Once a beatmap has accumulated 5 Hype, the beatmap is eligible for Nomination. This works as it did with the previous modding system. The beatmap is required to have 2 nominations by the [Beatmap Nominators](/wiki/Beatmap_Nominators) to move into the [qualified](/wiki/qualified) state.
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: has accumulated -> accumulates

File: wiki/Beatmap_Discussion/en.md [hunk 5]
```diff
@@ -134,30 +140,49 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
 - Mark as Resolved: The creator may select this when the issue has been addressed and either accepted or denied.
 - Reopen a Discussion: If the submission needs further discussion, the creator or mod submitter may reopen the submission by responding and unchecking Resolved.
 
-## Earning Kudosu!
+## Hype, Nomination, and Kudosu!
+
+### Hype
 
-*Note: You get ten votes every hour.*
+Hype is used to help push beatmaps forward to nomination; similar to how Star Priority was previously used. Once a beatmapset has received 5 Hype, it becomes eligible for Nomination. Please note the following:
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: has received -> receives

File: wiki/Beatmap_Discussion/en.md [hunk 6]
```diff
@@ -134,30 +140,49 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
...
+
+- Each user has a limited supply of hype (5 per month to start).
+  - You will regain a hype 30 days after that one hype was used.
+- Hype can only be used after playing through a difficulty from a beatmapset.
+
+### Nomination
+
+Nomination is similar to the previous modding system. A beatmap must receive 2 nominations from two [Beatmap Nominators](/wiki/Beatmap_Nominators) before being automatically moved to the qualification stage. Beatmaps must have all Suggestions and Problems resolved before it can be ranked. BN/QAT now can reset nominations by posting a problem in the discussion field.
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: it can be ranked -> it can get ranked

File: wiki/Beatmap_Discussion/en.md [hunk 7]
```diff
@@ -134,30 +140,49 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
...
+
+### Kudosu!
 
 ![](img/thumbs.jpg "Thumbs up for Kudosu!")
 
-Modders can now earn Kudosu! by having a thumbs up on their mod. You can (and probably should) go through old mods on your beatmaps and give them a thumbs up to support the modders before this feature was implemented.
+Giving a thumbs up to a suggestion or problem in the discussion field will earn the modder Kudosu!. The amount of thumbs up received determines how much Kudosu! the modder earns. Any user besides the poster can thumbs up any post. *Note: You get 60 votes every hour.*
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: will earn -> will give
Giving a thumb up
earns -> gets
can thumb up

File: wiki/Beatmap_Discussion/en.md [hunk 8]
```diff
@@ -134,30 +140,49 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
...
+
+Currently, these are the thresholds for earning an amount of Kudosu!:
+
+- 1 thumbs up = 1 Kudosu!
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: 1 thumb up

File: wiki/Beatmap_Discussion/en.md [hunk 9]
```diff
@@ -134,30 +140,49 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
...
 
-- Submit your beatmap through Stable (Latest), Beta, or Cutting Edge to use Beatmap Discussion. Fallback submits to the old system.
+- Submit your beatmap through Stable (Latest), Beta, or Cutting Edge to use Beatmap Discussion. Fallback submits to the previous system.
 - Choose the correct mode and difficulty you wish to mod!
 - Select the proper section you are submitting your mod to.
 - **Only post one issue at a time.** Do not submit multiple issues in a single post!
 - The Timeline section requires at least one timestamp before you can submit your post.
 - Check to see if the issue has already been addressed when the similar post warning pops up.
 - Do you like the beatmap? Give it some Praise in the General (All Difficulties) section to increase the amount of Hype on the Hype Train. This helps the beatmap move closer to nomination.
+- Be sure to vote on suggestions or problems, as they now award Kudosu! to creators.
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: capitalize suggestions and problems or decapitalize above

File: wiki/Beatmap_Discussion/en.md [hunk 10]
```diff
@@ -171,21 +196,17 @@ Your post will be submitted! You can restart at step 3 if you wish to continue m
...
 
-### Is Kudosu! available in Beatmap Discussion?
-
-Kudosu! has been recently implemented in the Beatmap Discussion! You can give modders Kudosu! by using the thumbs up emote on discussions that were marked as Suggestions or Problems.
-
-### How are beatmaps nominated?
+### How do I edit my beatmap's description with Beatmap Discussion?
 
-Beatmap Discussion follows the same procedure as its predecessor. 2 Beatmap Nominators are required to nominate the Meatmapset for qualification. Star Priority has been replaced with Hype and requires 12 Hype before it can be nominated.
+You can edit your beatmap description on the new site by either submitting your map and editing it like the past, or you may simply edit it at any time by clicking on the edit icon in the top right of the description field in the beatmapset's page.
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: your beatmap's description
like in the past

File: wiki/Beatmap_Discussion/en.md (hunk 1 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: is this okay?
``The number that appears besides the difficulty name are the number of pending discussions it currently has that needs to be resolved.``

Comment (TPGPL) [reply wiki/Beatmap_Discussion/en.md]: the number **are**?

Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: can you explain further on 
> the number **are**?

File: wiki/Beatmap_Discussion/en.md (hunk 2 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: > Hype must be done in the General (All Difficulties) section to add Hype -> If you want to add Hype, it must be done in the General (All Difficulties) section.
OK

---
>"must be done in the General section" "it wont work through the General tab" what the fuck
why is a "beatmap" capitalized at the end of the line
what, can you explain what you suggested?

File: wiki/Beatmap_Discussion/en.md (hunk 3 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: could you explain why it should be `at the time` instead of `at this time`?

File: wiki/Beatmap_Discussion/en.md (hunk 7 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: > Giving a thumb up

could you explain why it should be `Giving a thumb up` instead of `Giving a thumbs up`? (I messaged this in the \#osu-wiki saying that using "thumbs up" just flows better when reading it)

---

> can thumb up

ditto

File: wiki/Beatmap_Discussion/en.md (hunk 8 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: ditto (https://github.com/ppy/osu-wiki/pull/933#discussion_r158563976)

File: wiki/Beatmap_Discussion/en.md (hunk 1 above)
Comment (TPGPL) [reply wiki/Beatmap_Discussion/en.md]: The number that appears besides the difficulty name is the number of pending discussions which have to get resolved.

You wrote "The number are"

File: wiki/Beatmap_Discussion/en.md (hunk 2 above)
Comment (TPGPL) [reply wiki/Beatmap_Discussion/en.md]: Those sentences exclude themselves.
The beatmap shouldnt be capitalized as it is on the end of the line.

File: wiki/Beatmap_Discussion/en.md (hunk 3 above)
Comment (TPGPL) [reply wiki/Beatmap_Discussion/en.md]: It currently appears to do nothing. 
Now it might be better

File: wiki/Beatmap_Discussion/en.md (hunk 7 above)
Comment (TPGPL) [reply wiki/Beatmap_Discussion/en.md]: Thumb up is singular I think

File: wiki/Beatmap_Discussion/en.md (hunk 5 above)
Comment (TPGPL) [reply wiki/Beatmap_Discussion/en.md]: + it will become

File: wiki/Beatmap_Discussion/en.md (hunk 1 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: I have yet to understand what you mean, I am pretty sure that that you spoted was even there. 😕

File: wiki/Beatmap_Discussion/en.md (hunk 2 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: sorry, I wanted clarification for the things in quotes, not the caps of beatmaps...
was that a suggestion or just pointing something out with no suggestion?

File: wiki/Beatmap_Discussion/en.md (hunk 5 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: what?

File: wiki/Beatmap_Discussion/en.md (hunk 2 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: disregard-- I found out what you had meant and combined the sentences together.

File: wiki/Beatmap_Discussion/en.md (hunk 1 above)
Comment (TPGPL) [reply wiki/Beatmap_Discussion/en.md]:
```is this okay?
The number that appears besides the difficulty name are the number of pending discussions it currently has that needs to be resolved.```
you wrote "The number (...) are"

File: wiki/Beatmap_Discussion/en.md [hunk 11]
```diff
@@ -72,19 +72,21 @@ This contains the creator who submitted the beatmapset, shows the submission dat
...
 ### Hype Train
 
-Hype train is similar to the old star priority system. A user can submit Praise to the beatmap and it will add to the beatmap's Hype Train.
+Hype train is similar to the previous star priority system. A user can submit Praise to the beatmap and it will add to the beatmap's Hype Train.
 
 > Adding praise ❤ will raise this beatmap's hype, making it more visible for nomination and ranking!
 
 *Note: the creator can't Hype up their own beatmap. Praise must be done in the General (All Difficulties) section to add Hype. It will not work through the General or the Timeline tabs.*
 
+If you want to add Hype, it must be done in the General (All Difficulties) tab, not the General or the Timeline tabs. Players are required to play through a difficulty in the beatmapset in order to add Hype to a beatmap.
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: I still dont understand this first sentence.

It must be done in the General tab, not the General tab.

File: wiki/Beatmap_Discussion/en.md [hunk 12]
```diff
@@ -126,38 +132,57 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
...
-## Earning Kudosu!
+## Hype, Nomination, and Kudosu!
+
+### Hype
 
-*Note: You get ten votes every hour.*
+Hype is used to help push beatmaps forward to nomination; similar to how Star Priority was previously used. Once a beatmapset receives 5 Hype, it becomes eligible for Nomination. Please note the following:
+
+- Each user has a limited supply of hype (5 per month to start).
+  - You will regain a hype 30 days after that one hype was used.
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: after one hype was used

File: wiki/Beatmap_Discussion/en.md [hunk 13]
```diff
@@ -126,38 +132,57 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
...
+- Hype can only be used after playing through a difficulty from a beatmapset.
+
+### Nomination
+
+Nomination is similar to the previous modding system. A beatmap must receive 2 nominations from two [Beatmap Nominators](/wiki/Beatmap_Nominators) before being automatically moved to the qualification stage. Beatmaps must have all Suggestions and Problems resolved before it can get ranked. BN/QAT now can reset nominations by posting a problem in the discussion field.
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: before it can go ranked

File: wiki/Beatmap_Discussion/en.md [hunk 14]
```diff
@@ -126,38 +132,57 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
...
 
-- Submit your beatmap through Stable (Latest), Beta, or Cutting Edge to use Beatmap Discussion. Fallback submits to the old system.
+- Submit your beatmap through Stable (Latest), Beta, or Cutting Edge to use Beatmap Discussion. Fallback submits to the previous system.
 - Choose the correct mode and difficulty you wish to mod!
 - Select the proper section you are submitting your mod to.
 - **Only post one issue at a time.** Do not submit multiple issues in a single post!
 - The Timeline section requires at least one timestamp before you can submit your post.
 - Check to see if the issue has already been addressed when the similar post warning pops up.
 - Do you like the beatmap? Give it some Praise in the General (All Difficulties) section to increase the amount of Hype on the Hype Train. This helps the beatmap move closer to nomination.
+- Be sure to vote on Suggestions or Problems, as they now award Kudosu! to creators.
```
Comment (TPGPL) [inline wiki/Beatmap_Discussion/en.md]: to modders

File: wiki/Beatmap_Discussion/en.md (hunk 11 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: image

Comment (TPGPL) [reply wiki/Beatmap_Discussion/en.md]: oh

Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: I'll make some changes (probably add code) on this line and with the note above, since they are about the same thing just in a note and outside of a note.

File: wiki/Beatmap_Discussion/en.md (hunk 12 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: I specifically choose to word it as `after that one` because it refers to the specific hype that was used at that time

will reword it to `You will regain a hype 30 days after it was used.`

File: wiki/Beatmap_Discussion/en.md (hunk 13 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: is there a reason for why it should be `go` instead of `get`? I think the current is just fine 😕

**Comment by Xinnoh** [comment]
Should add something about the ability to sort by time posted

File: wiki/Beatmap_Discussion/en.md [hunk 15]
```diff
@@ -72,19 +72,21 @@ This contains the creator who submitted the beatmapset, shows the submission dat
 
 ### Watch/Unwatch
 
-This replaces beatmap subscription. You can track the beatmaps' status and open issues. This page is access via the website's Home page by clicking on `modding watchlist` located near the top of the page.
+This replaces beatmap subscription. You can track the beatmaps' status and open issues. This page is access via the website's Home page by clicking on [`modding watchlist` link](https://osu.ppy.sh/beatmapsets/watches) located near the top of the page.
 
 ### Hype Train
 
-Hype train is similar to the old star priority system. A user can submit Praise to the beatmap and it will add to the beatmap's Hype Train.
+Hype train is similar to the previous star priority system. A user can submit Praise to the beatmap and it will add to the beatmap's Hype Train.
```
Comment (Xinnoh) [inline wiki/Beatmap_Discussion/en.md]: Submitting praise no longer gives hype, they have to click the hype button

File: wiki/Beatmap_Discussion/en.md [hunk 16]
```diff
@@ -72,19 +72,21 @@ This contains the creator who submitted the beatmapset, shows the submission dat
...
 > Adding praise ❤ will raise this beatmap's hype, making it more visible for nomination and ranking!
 
 *Note: the creator can't Hype up their own beatmap. Praise must be done in the General (All Difficulties) section to add Hype. It will not work through the General or the Timeline tabs.*
 
+If you want to add Hype, it must be done in the General (All Difficulties) tab, not the General or the Timeline tabs. Players are required to play through a difficulty in the beatmapset in order to add Hype to a beatmap.
+
 ### Nomination
 
-Once a beatmap has accumilated 12 Hype, the beatmap is eligible for Nomination. This works as it did with the old modding system. The beatmap is required to have 2 nominations by the [Beatmap Nominators](/wiki/Beatmap_Nominators) to move into the qualified state.
+Once a beatmap accumulates 5 Hype, the beatmap is eligible for Nomination. This works as it did with the previous modding system. The beatmap is required to have 2 nominations by the [Beatmap Nominators](/wiki/Beatmap_Nominators) to move into the [qualified](/wiki/qualified) state.
```
Comment (Xinnoh) [inline wiki/Beatmap_Discussion/en.md]: accumulates at least 5

File: wiki/Beatmap_Discussion/en.md (hunk 13 above)
Comment (RaikireHiuduo) [reply wiki/Beatmap_Discussion/en.md]: If the beatmap need to stay in qualification stage for a while before being ranked, then it must be mentioned rather than glanced over.

Same opinion with MegaApplePi.

File: wiki/Beatmap_Discussion/en.md [hunk 17]
```diff
@@ -111,13 +113,17 @@ This is where you type your mod. If you use the Timeline section, you must inclu
...
 ### Mod type
 
 ![](img/mod-type.jpg)
 
-This is where you submit your mod. You can choose between Praise, Suggestion, or Problem. Each is fairly self-explanatory. Clicking on this will submit your mod. If you made any mistake or need to make an edit to your mod, find the post in the Discussion section.. You can add Hype to the beatmap by submitting Praise in the General (All Difficulties) section.
+This is where you submit your mod. You can choose between Praise, Suggestion, or Problem. Each is fairly self-explanatory. **Clicking on this will submit your mod. If you made any mistake or need to make an edit to your mod, find the post in the Discussion section.** You can add Hype to the beatmap by submitting Praise in the General (All Difficulties) section.
+
+### Pin
+
+A blue button with a pin icon. It currently appears to do nothing.
```
Comment (Xinnoh) [inline wiki/Beatmap_Discussion/en.md]: https://i.imgur.com/UDDuupS.gifv

File: wiki/Beatmap_Discussion/en.md [hunk 18]
```diff
@@ -126,38 +132,57 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
 ### Functions
 
 - `Collapse all`/`Expand all` and the up arrow: These will show/hide posts.
-- Thumbs Up/Down: Like/Dislike the mod. Everyone but the mod submitter can use this. **Giving a thumbs up will now give the mod submitter [Kudosu!](/wiki/Kudosu!) on suggestions or problems.**
+- Thumbs Up/Down: Like/Dislike the mod. Everyone but the mod submitter can use this. **Giving a thumbs up will now give the mod submitter [Kudosu!](/wiki/Kudosu!) on Suggestions or Problems.**
```
Comment (Xinnoh) [inline wiki/Beatmap_Discussion/en.md]: Only nominators/qat/gmt can thumbs down

File: wiki/Beatmap_Discussion/en.md [hunk 19]
```diff
@@ -126,38 +132,57 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
...
 - Timeline: Shows the timestamp the issue addresses. Only available in the Timeline section.
 - CREATOR icon: Displays responses/submissions from the user who made the beatmapset.
 - permalink, edit, delete: Permalink grabs a direct link to the post. Edit allows the submitter to edit their submission. Delete allows the submitter to remove their submission.
 - Respond: Reply to the submission. Pressing Enter or clicking Reply afterwards will submit your response.
 - Mark as Resolved: The creator may select this when the issue has been addressed and either accepted or denied.
```
Comment (Xinnoh) [inline wiki/Beatmap_Discussion/en.md]: The creator of the mod can also resolve

File: wiki/Beatmap_Discussion/en.md [hunk 20]
```diff
@@ -126,38 +132,57 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
...
 
-## Earning Kudosu!
+## Hype, Nomination, and Kudosu!
+
+### Hype
 
-*Note: You get ten votes every hour.*
+Hype is used to help push beatmaps forward to nomination; similar to how Star Priority was previously used. Once a beatmapset receives 5 Hype, it becomes eligible for Nomination. Please note the following:
+
+- Each user has a limited supply of hype (5 per month to start).
```
Comment (Xinnoh) [inline wiki/Beatmap_Discussion/en.md]: 3 hype per week .-.

File: wiki/Beatmap_Discussion/en.md [hunk 21]
```diff
@@ -126,38 +132,57 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
...
+  - You will regain a hype 30 days after that one hype was used.
+- Hype can only be used after playing through a difficulty from a beatmapset.
```
Comment (Xinnoh) [inline wiki/Beatmap_Discussion/en.md]: this is not implemented and won't be for a while

File: wiki/Beatmap_Discussion/en.md (hunk 13 above)
Comment (Xinnoh) [reply wiki/Beatmap_Discussion/en.md]: be or become works best

Comment (Xinnoh) [inline wiki/Beatmap_Discussion/en.md]: BN/QAT can reset nominations
It's not like a new feature, it was just missing before.

File: wiki/Beatmap_Discussion/en.md [hunk 22]
```diff
@@ -126,38 +132,57 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
...
+### Nomination
+
+Nomination is similar to the previous modding system. A beatmap must receive 2 nominations from two [Beatmap Nominators](/wiki/Beatmap_Nominators) before being automatically moved to the qualification stage. Beatmaps must have all Suggestions and Problems resolved before it can get ranked. BN/QAT now can reset nominations by posting a problem in the discussion field.
+
+### Kudosu!
 
 ![](img/thumbs.jpg "Thumbs up for Kudosu!")
 
-Modders can now earn Kudosu! by having a thumbs up on their mod. You can (and probably should) go through old mods on your beatmaps and give them a thumbs up to support the modders before this feature was implemented.
+Giving a thumbs up to a suggestion or problem in the discussion field will give the modder Kudosu!. The amount of thumbs up received determines how much Kudosu! the modder gets. Any user besides the poster can thumbs up any post. *Note: You get 60 votes every hour.*
```
Comment (Xinnoh) [inline wiki/Beatmap_Discussion/en.md]: Currently worded like you should do 60 votes per hour, should say it's limited or capped to 60 votes per hour
Also, BN/QAT don't have this limit.

File: wiki/Beatmap_Discussion/en.md [hunk 23]
```diff
@@ -126,38 +132,57 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
...
+Currently, these are the thresholds for earning a certain amount of Kudosu!:
+
+- 1 thumbs up = 1 Kudosu!
+- 2 thumbs up = 2 Kudosu!
+- 5 thumbs up = 3 Kudosu!
 
 ## Reminders
 
-- Submit your beatmap through Stable (Latest), Beta, or Cutting Edge to use Beatmap Discussion. Fallback submits to the old system.
+- Submit your beatmap through Stable (Latest), Beta, or Cutting Edge to use Beatmap Discussion. Fallback submits to the previous system.
```
Comment (Xinnoh) [inline wiki/Beatmap_Discussion/en.md]: Not needed since fallback is gone

File: wiki/Beatmap_Discussion/en.md [hunk 24]
```diff
@@ -126,38 +132,57 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
...
 - Choose the correct mode and difficulty you wish to mod!
 - Select the proper section you are submitting your mod to.
 - **Only post one issue at a time.** Do not submit multiple issues in a single post!
 - The Timeline section requires at least one timestamp before you can submit your post.
 - Check to see if the issue has already been addressed when the similar post warning pops up.
 - Do you like the beatmap? Give it some Praise in the General (All Difficulties) section to increase the amount of Hype on the Hype Train. This helps the beatmap move closer to nomination.
```
Comment (Xinnoh) [inline wiki/Beatmap_Discussion/en.md]: needs rewording since praise doesn't give hype

**Comment by MegaApplePi** [comment]
@Xinnoh alright, I just didn't realize that the web-devs already implemented it (more so that the thread wasn't updated yet ¯\\\_(ツ)\_/¯)

File: wiki/Beatmap_Discussion/en.md [hunk 25]
```diff
@@ -171,21 +196,17 @@ Your post will be submitted! You can restart at step 3 if you wish to continue m
...
 
-### Is Kudosu! available in Beatmap Discussion?
-
-Kudosu! has been recently implemented in the Beatmap Discussion! You can give modders Kudosu! by using the thumbs up emote on discussions that were marked as Suggestions or Problems.
-
-### How are beatmaps nominated?
+### How do I edit my beatmap's description with Beatmap Discussion?
 
-Beatmap Discussion follows the same procedure as its predecessor. 2 Beatmap Nominators are required to nominate the Meatmapset for qualification. Star Priority has been replaced with Hype and requires 12 Hype before it can be nominated.
+You can edit your beatmap's description on the new site by either submitting your map and editing it like in the past, or you may simply edit it at any time by clicking on the edit icon in the top right of the description field in the beatmapset's page.
```
Comment (Xinnoh) [inline wiki/Beatmap_Discussion/en.md]: bit redundant since it's the same thing a couple lines up

File: wiki/Beatmap_Discussion/en.md [hunk 26]
```diff
@@ -171,21 +196,17 @@ Your post will be submitted! You can restart at step 3 if you wish to continue m
...
 
 ### Will I be able to mod with the older system after Beatmap Discussion is released?
 
 You should mod with the system the creator has chosen. If they focus on Beatmap Discussion, mod with Beatmap Discussion. Otherwise your mods may go unnoticed.
 
 ### I don't like Beatmap Discussion! How do go back to the older system?
```
Comment (Xinnoh) [inline wiki/Beatmap_Discussion/en.md]: how do i

File: wiki/Beatmap_Discussion/en.md [hunk 27]
```diff
@@ -72,19 +72,21 @@ This contains the creator who submitted the beatmapset, shows the submission dat
 
 ### Watch/Unwatch
 
-This replaces beatmap subscription. You can track the beatmaps' status and open issues. This page is access via the website's Home page by clicking on `modding watchlist` located near the top of the page.
+This replaces beatmap subscription. You can track the beatmaps' status and open issues. This page is access via the website's Home page by clicking on [`modding watchlist` link](https://osu.ppy.sh/beatmapsets/watches) located near the top of the page.
```
Comment (Xinnoh) [inline wiki/Beatmap_Discussion/en.md]: this page is accessed

File: wiki/Beatmap_Discussion/en.md [hunk 28]
```diff
@@ -72,19 +72,21 @@ This contains the creator who submitted the beatmapset, shows the submission dat
...
 
 ### Hype Train
 
-Hype train is similar to the old star priority system. A user can submit Praise to the beatmap and it will add to the beatmap's Hype Train.
+Hype train is similar to the previous star priority system. A user can submit Praise to the beatmap and it will add to the beatmap's Hype Train.
 
 > Adding praise ❤ will raise this beatmap's hype, making it more visible for nomination and ranking!
 
 *Note: the creator can't Hype up their own beatmap. Praise must be done in the General (All Difficulties) section to add Hype. It will not work through the General or the Timeline tabs.*
```
Comment (Xinnoh) [inline wiki/Beatmap_Discussion/en.md]: not really needed since the hype button was removed for the creator, this was written when it was still there for the creator.

File: wiki/Beatmap_Discussion/en.md (hunk 11 above)
Comment (Xinnoh) [inline wiki/Beatmap_Discussion/en.md]: Playing through a diff before hyping is an unimplemented feature

File: wiki/Beatmap_Discussion/en.md [hunk 29]
```diff
@@ -126,38 +132,57 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
 ### Functions
 
 - `Collapse all`/`Expand all` and the up arrow: These will show/hide posts.
-- Thumbs Up/Down: Like/Dislike the mod. Everyone but the mod submitter can use this. **Giving a thumbs up will now give the mod submitter [Kudosu!](/wiki/Kudosu!) on suggestions or problems.**
+- Thumbs Up/Down: Like/Dislike the mod. Everyone but the mod submitter can use this. **Giving a thumbs up will now give the mod submitter [Kudosu!](/wiki/Kudosu!) on Suggestions or Problems.**
 - Timeline: Shows the timestamp the issue addresses. Only available in the Timeline section.
 - CREATOR icon: Displays responses/submissions from the user who made the beatmapset.
```
Comment (Xinnoh) [inline wiki/Beatmap_Discussion/en.md]: isn't it "mapper"

File: wiki/Beatmap_Discussion/en.md [hunk 30]
```diff
@@ -126,38 +132,57 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
...
 - permalink, edit, delete: Permalink grabs a direct link to the post. Edit allows the submitter to edit their submission. Delete allows the submitter to remove their submission.
 - Respond: Reply to the submission. Pressing Enter or clicking Reply afterwards will submit your response.
```
Comment (Xinnoh) [inline wiki/Beatmap_Discussion/en.md]: Respond/Reply, since  discussions with 1+ comments change to that for some reason 🤔

File: wiki/Beatmap_Discussion/en.md (hunk 11 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: will comment out for now then

File: wiki/Beatmap_Discussion/en.md (hunk 17 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: oooooooooh lol

File: wiki/Beatmap_Discussion/en.md (hunk 29 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: nope, creator
at some point, ppy said to use "creator" instead of "mapper"

Comment (Xinnoh) [reply wiki/Beatmap_Discussion/en.md]: 🤔 
screen shot 2017-12-28 at 11 45 07 pm

File: wiki/Beatmap_Discussion/en.md (hunk 21 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: will comment out

File: wiki/Beatmap_Discussion/en.md (hunk 25 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: removed, I'm running out of time, so I'll have good faith and assume so

File: wiki/Beatmap_Discussion/en.md (hunk 24 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: reworded to `Do you like the beatmap? Give your hype to help the beatmap move closer to nomination!`

File: wiki/Beatmap_Discussion/en.md (hunk 22 above)
Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: whoops, missed this... is it just BN and QAT, not GMT?

Comment (MegaApplePi) [reply wiki/Beatmap_Discussion/en.md]: oh wait, never mind... will reword to `*Note: You are limited to 60 votes per hour, while BN and QAT members do not have this limit.*`
